website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -40,68 +48,323 @@ Authentication uses a dbt Cloud [service account tokens](/docs/dbt-cloud-apis/se
 
 ```graphql
 queryableGranularities(
-environmentId: Int!
-metrics: [String!]!
+  environmentId: BigInt!
+  metrics: [String!]!
 ): [TimeGranularity!]!
 ```
 
+You can also get queryable granularities for all other dimensions using the `dimensions` call:
+
+```graphql
+{
+  dimensions(environmentId: BigInt!, metrics:["order_total"]) {
+    name
+    queryableGranularities # --> ["DAY", "WEEK", "MONTH", "QUARTER", "YEAR"]
+  }
+}
+```
+
+You can also optionally access it from the metrics endpoint:
+
+```graphql
+{
+  metrics(environmentId: BigInt!) {
+    name
+    dimensions {
+      name
+      queryableGranularities
+    }
+  }
+}
+```
+
+**Fetch measures**
+```graphql
+{
+  measures(metrics: ["order_total"], environmentId: BigInt!) {
+    name
+    aggTimeDimension
+  }
+}
+```
+`aggTimeDimension` tells you the name of the dimension that maps to `metric_time` for a given measure. You can also query `measures` from the `metrics` endpoint, which allows you to see what dimensions map to `metric_time` for a given metric:
+```graphql
+{
+  metrics(environmentId: BigInt!) {
+    measures {
+      name
+      aggTimeDimension
+    }
+  }
+}
+```
+
 **Fetch available metrics given a set of a dimensions**
 
 ```graphql
 metricsForDimensions(
-environmentId: Int!
-dimensions: [String!]!
+  environmentId: BigInt!
+  dimensions: [String!]!
 ): [Metric!]!
 ```
 
-**Fetch dimension values for metrics and a given dimension**
+**Create Dimension Values query**
 
 ```graphql
-dimensionValues(
-environmentId: Int!
-metrics: [String!]!
-dimension: String!
-```
 
-### Metric value query parameters
+mutation createDimensionValuesQuery(
+  environmentId: BigInt!
+  metrics: [String!]
+  groupBy: [String!]!
+): CreateDimensionValuesQueryResult!
+
+```
 
-The mutation is `createQuery`. The parameters are as follows:
+**Create Metric query**
 
 ```graphql
 createQuery(
-environmentId: Int!
+environmentId: BigInt!
 metrics: [String!]!
-dimensions: [String!] = null
+group_by: [String!] = null
 limit: Int = null
-startTime: String = null
-endTime: String = null
-where: String = null
+where: [String] = null
 order: [String!] = null
-): String
+): CreateQueryResult
+```
+
+**Fetch query result**
+```graphql
+query(
+  environmentId: BigInt!
+  queryId: String!
+): QueryResult!
+```
+
+**Metric Types**
+
+```graphql
+Metric {
+	name: String!
+	description: String
+	type: MetricType!
+	typeParams: MetricTypeParams!
+	filter: WhereFilter
+	dimensions: [Dimension!]!
+  queryableGranularities: [TimeGranularity!]!
+}
+```
+
+```
+MetricType = [SIMPLE, RATIO, CUMULATIVE, DERIVED]
+```
+
+**Metric Type Parameters**
+
+```graphql
+MetricTypeParams {
+	measure: MetricInputMeasure
+	inputMeasures: [MetricInputMeasure!]!
+	numerator: MetricInput
+	denominator: MetricInput
+	expr: String
+	window: MetricTimeWindow
+	grainToDate: TimeGranularity
+	metrics: [MetricInput!]
+}
+```
+
+
+**Dimension Types**
+todo: add info on granularity when we expose it
```

Comment:
hey @WilliamDee and @courtneyholcomb, this section seems like it's missing granularity info -  is that something either one of you can add please?

Reply:
Oh woops, forgot to remove this, this is done right down below with the addition of
```
queryableGranularities: [TimeGranularity!]!
```